wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md

```diff
@@ -40,7 +40,7 @@ The following are modding traits expected of Beatmap Nominators. Failure to disp
 `m` = Minimum expected number of individual beatmaps modded in a month\
 `M` = Total achieved number of individual beatmaps modded in a month
 
-- [The website](http://bn.mappersguild.com/bnapps) has a tool for calculating mod score on its main page.
+- [The website](http://bn.mappersguild.com/bnapps) has a tool for calculating `Mod Score` on its main page.
```

Comment:
Is this how we use backticks in the wiki? Generally it's only used for variable names.. so something like bold or quotations may work better for terms like this?

Reply:
The rest of the article formatted it this way, I personally have no idea or opinion on what is correct here

Reply:
Would need to check the wiki style guide - should be fixed to match what we do everywhere else.

Reply:
https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#keywords-and-commands could apply? I don't know - it is used as a keyword